README.md

```diff
@@ -39,7 +39,12 @@ devtools::install_github("UptakeOpenSource/uptasticsearch")
 
 ### Python <a name="pythoninstallation"></a>
 
-We plan to release a Python implementation of this functionality, but that is not available at this time. Check back often!
+To install the python package:
```

Comment:
Can you add language about PyPi to be explicit? Like

*This package is not currently available on PyPi. To build the development version of the package from source, clone the repo and then do the following:*